tensormap-client/src/components/DragAndDropCanvas/CustomNodes/FlattenNode/FlattenNode.js

```diff
@@ -0,0 +1,43 @@
+import {Handle,Position,useStoreApi,useReactFlow} from 'reactflow'
+
+import './FlattenNode.css'
+
+function FlattenNode({data,id }) {
+    const {setNodes} = useReactFlow();
+    const store = useStoreApi();
+    const onChange = (evt) => {
+      const { nodeInternals } = store.getState();
+      const { name, value } = evt.target;
+      setNodes(
+        Array.from(nodeInternals.values()).map((node) => {
+          if (node.id === id) {
+            node.data = {
+              ...node.data,
+              params: {
+                ...node.data.params,
+                [name]: Number(value),
+              },
+            };
+          }
+          return node;
+        })
+      );
+    };
+
+
+  
+    return (
+      <div className="flatten-node">
+        <Handle type="source" position={Position.Right} isConnectable={true} id={`${id}_out`}  />
+        <div className='node-header'>Flatten Node</div>
+        <label htmlFor="text">Dim X</label>
+        <input id="dim-x" name="dim-x" type='number' min="0" onChange={onChange} className="nodrag" value={data.params["dim-x"]}/>
+        <br/>
+        <label htmlFor="text">Dim Y</label>
+        <input id="dim-y" name="dim-y" type='number' min="0" onChange={onChange} className="nodrag" value={data.params["dim-y"]} />
```

Comment:
Can we also give a placeholder to these input fields?

Reply:
Add on to placeholder point, 
I see in the input field, we have given `onChange` attribute. The callback function named `onChange` sounds "general" input's onChange. Instead, to give a generic name, can we think of a better name? What are your thoughts?